### PR TITLE
Include pod namespace in the pod name

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: server/sandbox.go [hunk 1]
```diff
@@ -59,8 +60,13 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 		return nil, fmt.Errorf("PodSandboxConfig.Name should not be empty")
 	}
 
+	namespace := req.GetConfig().GetMetadata().GetNamespace()
+	if namespace == "" {
+		namespace = podDefaultNamespace
```
Comment (mrunalp) [inline server/sandbox.go]: Sure, I'll fix this up.

File: server/sandbox.go [hunk 2]
```diff
@@ -39,12 +40,12 @@ func (s *sandbox) removeContainer(c *oci.Container) {
 	s.containers.Delete(c.Name())
 }
 
-func (s *Server) generatePodIDandName(name string) (string, string, error) {
+func (s *Server) generatePodIDandName(name string, namespace string) (string, string, error) {
```
Comment (mrunalp) [inline server/sandbox.go]: Sure.

**Comment by mrunalp** [comment]
@runcom Fixed.

**Comment by runcom** [comment]
LGTM

**Pull request merged**